config/variables.go

```diff
@@ -0,0 +1,172 @@
+// Package config defines the default configuration and binds to environment variables
+package config
+
+import (
+	"github.com/flashbots/mev-boost-relay/common"
+	"github.com/spf13/viper"
+)
+
+const (
+	// Default values
+	DefaultNetwork     = ""
+	DefaultRedisURI    = "localhost:6379"
+	DefaultPostgresDSN = ""
+	DefaultLogJSON     = false
+	DefaultLogLevel    = "info"
+
+	APIDefaultListenAddr         = "localhost:9062"
+	APIDefaultBlockSim           = "http://localhost:8545"
+	APIDefaultSecretKey          = ""
+	APIDefaultLogTag             = ""
+	APIDefaultLogVersion         = false
+	APIDefaultPprofEnabled       = false
+	APIDefaultInternalAPIEnabled = false
+	APIDefaultDebug              = false
+
+	WebsiteDefaultListenAddr        = "localhost:9060"
+	WebsiteDefaultShowConfigDetails = false
+	WebsiteDefaultLinkBeaconchain   = "https://beaconcha.in"
+	WebsiteDefaultLinkEtherscan     = "https://etherscan.io"
+	WebsiteDefaultRelayURL          = ""
+	WebsiteDefaultPubkeyOverride    = ""
+
+	// Common: Api, HouseKeeper & Website
+	Network        = "Network"
```

Comment:
I wonder if we should prefix them with `Key` -- like `KeyNetwork` to make it clear it's the key, and not the network, when using it.

Reply:
Sure, I can give it a try